foundations/html_css/block-and-inline.md

```diff
@@ -17,6 +17,8 @@ Most of the elements that you have learned about so far are block elements.  In
 
 Inline elements, however, do not start on a new line. They appear in line with whatever elements they are placed beside. A clear example of an inline element is a link, or `<a>` tag. If you stick one of these in the middle of a paragraph of text, it will behave like a part of the paragraph. ([Like this...](https://www.youtube.com/watch?v=dQw4w9WgXcQ)) The link's text will sit alongside other words in that paragraph. Additionally, padding and margin behave differently on inline elements. In general, you do not want to try to put extra padding or margin on inline elements.
 
+#### The middle ground `inline-block`
```

Comment:
Can you explain why you think we need a new heading in this location?

In addition, we do not put code blocks in our headings. Please see our layout style guide
https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#no-code-snippets

Reply:
Ok. That does make sense. I don't think it is an essential change, but I understand your point of view.